.github/workflows/tests.yml

```diff
@@ -189,6 +189,32 @@ jobs:
         name: system-test-screenshots
         path: tmp/screenshots/*
 
+  screenshots_link:
+    name: Comment screenshots link
+    needs: [system-tests, comment-system-tests]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v5
+        if: github.event_name == 'pull_request'
+        with:
+          script: |
+            const artifacts = await github.rest.actions.listWorkflowRunArtifacts({
```

Comment:
This won't work until the run is completed. We have to find another way or give up this idea until it's resolved! 

https://github.community/t/synchronous-update-of-the-artifacts-when-queried-through-api/17520/2